### PR TITLE
Fix google custom search bug

### PR DESCRIPTION
**Pull request by codyrobert**
A portion of the time when loading `/search/`, no results would be returned and an error would display in the console such as:

```
Uncaught TypeError: google.search.Search.apiaryXXX is not a function
```

The problem was the google custom search script was being loaded twice, via `master.js` and `search.js`. I've resolved this issue by removing the instance in `master.js`.

---

Thanks,
Cody Baker
Flickerbox, Inc.

### CODE REVIEW

**Comment by johngronberg-okta** [comment]
@rchild-okta can you take a look?

File: assets/js/search.js [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
-	gcse.async = true;
-	gcse.src = (document.location.protocol == 'https:' ? 'https:' : 'http:') +
-	    '//cse.google.com/cse.js?cx=' + cx;
-	var s = document.getElementsByTagName('script')[0];
-	s.parentNode.insertBefore(gcse, s);
-    }
-    initSearch();
+(function($) {
+	
+	$(document).ready(function() {
```
Comment (rchild-okta) [inline assets/js/search.js]: nit, just use:
```javascript
$(function () {
  // code
});
```

No need for the $(document).ready() part, and that pattern is deprecated in newer versions of jquery.

File: assets/js/search.js [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
+		gcse.type = 'text/javascript';
+		gcse.async = true;
+		gcse.src = document.location.protocol + '//cse.google.com/cse.js?cx=' + cx;
+	
+		var s = document.getElementsByTagName('script')[0];
+		s.parentNode.insertBefore(gcse, s);
+    
+    });
+
+})(jQuery);
```
Comment (rchild-okta) [inline assets/js/search.js]: nit: newline

**Comment by zdevil** [comment]
Hi all,

I see that the changes were approved.
Is there anything we can help with to make this live? The search is broken and we would like to fix that asap.

Thanks a lot.
Hamza.

**Comment by johngronberg-okta** [comment]
@zdevil @codyrobert  we need a those couple of fixes that @rchild-okta mentioned. Then we can merge and deploy!

**Comment by codyrobert** [comment]
@johngronberg-okta Thanks - formatting fixed.

Cody

**Pull request merged**